CODEOWNERS

```diff
@@ -188,7 +188,7 @@ include/toolchain/                       @andrewboie @andyross
 include/zephyr.h                         @andrewboie @andyross
 kernel/                                  @andrewboie @andyross
 lib/gui/                                 @vanwinkeljan
-lib/posix/                               @ramakrishnapallala @nniranjhana @pfalcon
+lib/posix/                               @pfalcon
```

Comment:
Well, there're regular comments and even PRs from those guys, and I'm not comfortable to be there all alone ;-).